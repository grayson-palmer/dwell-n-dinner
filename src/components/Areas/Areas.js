import React from 'react';
import './Areas.scss';
import { Link } from 'react-router-dom';

const Areas = props => {
  const areaCards = props.areaSpecs.map(area => {
    return (
      <Link className="area-card">
        <h2>{area.name} - <i>{area.area}</i></h2>
        <p>{area.location}</p>
        <p>{area.about}</p>
      </Link>
    );
  });

  return <section className="Areas-container">{areaCards}</section>;
};

export default Areas;
