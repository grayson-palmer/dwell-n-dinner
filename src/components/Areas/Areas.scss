.area-card {
  border: .5rem solid black;
  border-radius: 1rem;
  height: 15em;
  text-decoration: none;
  margin: auto;
  padding: 1em;
  width: 20em;
}

.areas-container {
  grid-template-columns: 1fr 1fr;
  grid-template-rows: 1fr 1fr;
  display: grid;
  grid-gap: .5em; 
}

