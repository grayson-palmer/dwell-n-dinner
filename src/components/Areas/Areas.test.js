import React from 'react';
import { shallow } from 'enzyme';
import Areas from './Areas';

describe('Areas', () => {
  it('', () => {
    
  })
})