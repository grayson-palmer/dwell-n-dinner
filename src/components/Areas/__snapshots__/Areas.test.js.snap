// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Areas should render the component 1`] = `
<section
  className="areas"
>
  <Link
    className="areas__card"
    id="area-undefined"
    onClick={[Function]}
    to="/areas/undefined"
  >
    <div
      className="areas__info"
    >
      <h2>
        River North
         - 
        <i>
          RiNo
        </i>
      </h2>
      <p>
        <i>
          location
        </i>
      </p>
      <p>
        blah, blah, blah
      </p>
    </div>
  </Link>
</section>
`;
